app/containers/SubsciptionScheduleContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import store from '../store';
+import {connect} from 'react-redux';
+
+
+function mapStateToProps(state){
+  let freqency = state.preferences.frequency;
+  let time = state.preferences.time;
+
+
+  return {
+    frequency, time 
+  }
+}
+  
+function mapDispatchToProps(dispatch){
+  
+}
+
+export default connect(
+  mapStateToProps, 
+  mapDispatchToProps)(PreferencesClass);
```

Comment:
What is PreferencesClass? Shouldn't it be the SubscriptionSchedule component? We just need to import it at the top and we should be set.